manifests/m/MigrosBank/EBanking/9.2.2812/MigrosBank.EBanking.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: MigrosBank.EBanking
+PackageVersion: 9.2.2812
+InstallerType: exe
+InstallerSwitches:
+  Silent: --silentUpdate

```

```suggestion
  Silent: --silent
```
Had an urge to see if this undocumented switch would work, and yes, it results in the unattended install that we're seeking.